### DISCUSSION TITLE
Question about global styles

### DISCUSSION THREAD

**Discussion by seloner** [chakra-ui/chakra-ui#2536] [Q&A] [answered]
Is there an easy way to remove all border radius from all Components using the extendTheme ? Do I need to just do it overwriting components ?

**Comment by ljosberinn** [comment, marked as answer]
You should be able to remove borderRadius by overriding `radii` within theme:
```js
extendTheme({
  radii: {
    none: "0",
    sm: "0",
    base: "0",
    md: "0",
    lg: "0",
    xl: "0",
    "2xl": "0",
    "3xl": "0",
    full: "0",
  }
})
```

**Reply by seloner** [reply]
But I want to keep the other options for border radius I only want to change the default border radius for every component

**Reply by ljosberinn** [reply]
then youll have to individually override those, I'm afraid, yes
